### DISCUSSION TITLE
Failed to merge upstream content

### DISCUSSION THREAD

**Discussion by mfuji09** [mdn/mdn-community#435] [Localization] [answered]
I can't update local `translated-content` repository from the `upstream/main` branch now.
When I try it via GitHub Desktop, an error raises.

![image](https://github.com/mdn/mdn-community/assets/11516302/451bdf0c-76af-4ea7-beaf-837653201b0f)

I guess it is due to the file whose filename includes the `:` character, which is banned from filenames in Windows.
Could you please fix it? @mdn/yari-content-ko

**Comment by hochan222** [comment, marked as answer]
Oh I'm so sorry. I'll fix it right away.

**Reply by hochan222** [reply]
@mfuji09 https://github.com/mdn/translated-content/pull/15034 I solved it. Have a nice day :)

**Reply by mfuji09** [reply]
@hochan222 Thank you for your fix!

**Comment by pransh15** [comment]
Closing as this was answered.
